src/Diagnostics/Roslyn/Core/Performance/EquatableAnalyzer.cs

```diff
@@ -94,18 +93,24 @@ private void AnalyzeSymbol(SymbolAnalysisContext context, INamedTypeSymbol objec
 
         private bool IsObjectEqualsOverride(IMethodSymbol methodSymbol, INamedTypeSymbol objectType)
         {
+            Debug.Assert(methodSymbol != null);
+            if (methodSymbol == null)
+            {
+                return false;
+            }
+
             if (!methodSymbol.IsOverride)
             {
                 return false;
             }
 
             if (methodSymbol.Parameters.Length != 1 ||
```

Comment:
Can't add `?.` to `Parameters` -- diagnostic is "`Can't apply ? operator to ImmutableArray`